packages/browser-client/src/Components/Layout.tsx

```diff
@@ -43,8 +44,14 @@ export default function Layout() {
 
   const layoutVariant = useBreakpointValue({
     base: (
-      <Tabs height="100%" width={'100%'} index={state.tabIndex} onChange={handleTabsChange}>
-        <TabList height="5%" width="100%" justifyContent={'space-around'}>
+      <Tabs width={'100%'} index={state.tabIndex} onChange={handleTabsChange}>
```

Comment:
Quick note that the simple chat tab is only visible in a mobile equivalent view.

Reply:
I reverted all of the browser layout changes -- they were just a part of the process for me figuring things out